.eslintrc.js

```diff
@@ -1,49 +1,63 @@
 module.exports = {
-    'env': {
-        'es6': true,
-        'node': true
+    env: {
+        es6: true,
+        node: true
     },
-    'extends': [
-        'airbnb',
-        'eslint:recommended',
-        'plugin:react/recommended'
-    ],
-    'parser': 'babel-eslint',
-    'parserOptions': {
-        'ecmaVersion': 2018,
-        'sourceType': 'module'
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaVersion: 2020,
+        sourceType: 'module'
     },
-    'rules': {
-        'react/forbid-prop-types': false,
-        'import/extensions': false,
-        'ter-indent': [
-            true,
-            4,
-            { 'SwitchCase': 1 }
+    extends: [
+        'plugin:react/recommended',
+        'plugin:@typescript-eslint/recommended'
+    ],
+    rules: {
+        '@typescript-eslint/explicit-module-boundary-types': 'off', // TODO: This needs reviewing
+        '@typescript-eslint/no-explicit-any': 'off', // TODO: This need reviewing
+        '@typescript-eslint/no-var-requires': 'off', // TODO: This need reviewing
+        '@typescript-eslint/no-empty-function': 'off', // TODO: This needs reviewing
+        '@typescript-eslint/no-unused-vars': 'off', // TODO: This needs reviewing
+        '@typescript-eslint/no-non-null-assertion': 'off', // TODO: This needs reviewing
+        '@typescript-eslint/ban-ts-comment': 'off', // TODO: This needs reviewing
+        'react/prop-types': 'off', // TODO: This needs reviewing
+        'react/no-children-prop': 'off', // TODO: This needs reviewing
```

Comment:
Have we got ticket(s) raised to review these?

Reply:
The epic HOCS-2430 has been created to handle all of these - individual task tickets have also been added and associated with the epic. These have also added to the comments in this file.